CMakeLists.txt

```diff
@@ -86,6 +86,27 @@ CONFIGURE_FILE(
 	"${nanoflann_BINARY_DIR}/nanoflann.pc" @ONLY IMMEDIATE )
 
 
+
+# Set relative install directories
+set(INSTALL_INCLUDE_DIR "include")
+set(INSTALL_PKGCONFIG_DIR "lib${LIB_SUFFIX}/pkgconfig")
+set(INSTALL_CMAKE_DIR "lib${LIB_SUFFIX}/cmake/nanoflann")
+set(INSTALL_COPYRIGHT_DIR "share/doc/libnanoflann-dev")
+
+
+# Generate the cmake config and cmake config-version file:
+SET(RELATIVE_INCLUDE_DIR "../../../include")
+SET(CONFIG_INCLUDE_DIRS "\${nanoflann_CMAKE_DIR}/${RELATIVE_INCLUDE_DIR}")
+
```

Comment:
Hi, and first of all, *thanks* for the proposal! It's a very good idea to help.

I'm not familiar with the CMake variable `${PROJECT_NAME}_CMAKE_DIR`... I think both `*_SOURCE_DIR` and `*_BINARY_DIR` are more standard, aren't they?